DatadogFlags/Tests/Models/FlagAssignmentsResponseTests.swift

```diff
@@ -153,4 +153,148 @@ final class FlagAssignmentsResponseTests: XCTestCase {
             )
         )
     }
+
+    func testDecodingWithBrokenFlags_healsFromInvalidVariationType() throws {
```

Comment:
suggestion: Match existing naming patterns

```suggestion
    func testDecodingFlagAssignmentsResponseWithUnknownVariationTypes() throws {
```